config/schemas/config.yaml

```diff
@@ -178,9 +264,21 @@ $defs:
       - 3d
     pattern: ^[0-9]+[hmsd]+$
     title: An amount of time
+    description: |
+      Time range
     type: string
   kubernetesTolerations:
+    title: Kubernetes Tolerations
```

Comment:
We should rework this to use the upstream definition, the same for `kubernetesTopologySpreadConstraints`.

Reply:
Agreed, just need to check if they fit our usecase.

Reply:
I can not find a upstream schema for `topologySpreadConstraints` :thinking: 
Im looking here https://kubernetesjsonschema.dev/v1.14.0/_definitions.json#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta. 
Are there any other places I should look for it?

Reply:
I've look at here https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/v3/api__v1_openapi.json#L7797 the kubernetes json schema project is quite out of date.